Assets/MixedRealityToolkit/Definitions/Devices/ControllerMappingLibrary.cs

```diff
@@ -201,6 +206,8 @@ public static class ControllerMappingLibrary
         /// <returns></returns>
         public static InputManagerAxis[] UnityInputManagerAxes => new[]
         {
+            new InputManagerAxis { Name = UP_DOWN, Gravity = 3, Dead = 0.001f, Sensitivity = 3, Snap = true, Invert = false, Type = InputManagerAxisType.KeyOrMouseButton, PositiveButton = "e", NegativeButton = "q" },
+            new InputManagerAxis { Name = UP_DOWN,  Dead = defaultDeadZone, Sensitivity = 1, Invert = false, Type = InputManagerAxisType.JoystickAxis, Axis = 3 },
```

Comment:
What are your thoughts on adding this to the end of the array, instead of the beginning? I'm curious if someone out there is currently using this array to index into position 0 (they expect AXIS_1) but since we're hijacking that existing position, they get broken.

It might be possible to do static analysis on others' code to see if they're using it that way but, if at all possible, at the end is safest (same reason why we add enum values at the end rather than the beginning even if they logically make sense at the beginning)

Reply:
No objection. This input axis feature is not very robust in general, but Unity is introducing a new input system anyway, so i would put more effort into it at this point.
[https://forum.unity.com/threads/input-system-update.508660/](https://forum.unity.com/threads/input-system-update.508660/)